tests/app/user/test_rest.py

```diff
@@ -62,15 +62,24 @@ def test_get_user(admin_request, sample_service, sample_organisation):
 
     assert fetched["id"] == str(sample_user.id)
     assert fetched["name"] == sample_user.name
-    assert fetched["mobile_number"] == sample_user.mobile_number
     assert fetched["email_address"] == sample_user.email_address
-    assert fetched["state"] == sample_user.state
     assert fetched["auth_type"] == SMS_AUTH_TYPE
+    assert fetched["current_session_id"] is None
+    assert fetched["failed_login_count"] == 0
+    assert fetched["email_access_validated_at"]
+    assert fetched["logged_in_at"] is None
+    assert fetched["mobile_number"] == sample_user.mobile_number
+    assert fetched["organisations"] == [str(sample_organisation.id)]
+    assert fetched["password_changed_at"]
     assert fetched["permissions"].keys() == {str(sample_service.id)}
+    assert sorted(fetched["permissions"][str(sample_service.id)]) == sorted(expected_permissions)
+    assert fetched["organisation_permissions"] == {str(sample_organisation.id): []}
+    assert fetched["platform_admin"] is False
     assert fetched["services"] == [str(sample_service.id)]
-    assert fetched["organisations"] == [str(sample_organisation.id)]
     assert fetched["can_use_webauthn"] is False
-    assert sorted(fetched["permissions"][str(sample_service.id)]) == sorted(expected_permissions)
+    assert fetched["state"] == sample_user.state
+    assert fetched["take_part_in_research"] is True
+    assert fetched["receives_new_features_email"] is True
```

Comment:
Optional: You could consider adding an assertion on here about the number of keys in `fetched`. That way if we added any additional properties to a user, it would fail this test and we would be prompted to update it

Reply:
Good idea, that's now added (you will need to re-approve)